cinema_service/settings.py

```diff
@@ -15,14 +15,12 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = ("django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+              "mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
Why you change this?

Reply:
I changed secret_key because flake8 says the line is too long